slim/slim.go

```diff
@@ -166,7 +162,18 @@ func (e *Engine) Load() error {
 		if err != nil {
 			return err
 		}
-		// tmpl.FuncMap(e.funcmap)
+
+		// Init func map
+		newFuncMap := make(slim.Funcs, len(e.funcMap))
+		for key, val := range e.funcMap {
+			slimFunc, ok := val.(slimFunc)
+			if !ok {
+				panic("slim: function must be compatible with slim.Func type. Slim does not support other types")
```

Comment:
Should we use panic, error or just continue?

Reply:
What do you think @gaby @ReneWerner87

Reply:
Panic seems to be okay

Reply:
> Panic seems to be okay

But this works inside of Render. Is it good idea to panic on requests?